src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.xib

```diff
@@ -42,7 +42,7 @@
                             <rect key="frame" x="0.0" y="0.0" width="414" height="678.66666666666663"/>
```

Comment:
You rounded a lot of numbers, shouldn't this `height="678.66666666666663"` not be changed to `679`?

Reply:
Xcode did that 👍